tests/lib.rs

```diff
@@ -2,6 +2,9 @@ extern crate bson;
 
 #[macro_use(add_to_doc, doc)]
 extern crate mongodb;
+extern crate rustc_serialize;
+extern crate nalgebra as na;
```

Comment:
Don't rename packages unless necessary - makes it trickier to dig out what 'na' means.